WCSim.mac

```diff
@@ -132,6 +132,8 @@
 #/mygen/vecfile inputvectorfile
 #/mygen/vecfile h2o.2km.001-009x3_G4.kin
 #/mygen/vecfile mu+.out
+# define units used for time in kin file
+#/mygen/time_unit "ns"
```

Comment:
Probably don't need the `"` here. If they are required, then that is confusing since all the other parameters don't need them, and so should be fixed